hera_mc/sys_handling.py

```diff
@@ -45,6 +45,39 @@ def cofa(self):
         cofa = self.geo.cofa()
         return cofa
 
+    def get_dubitable_list(self, date='now', return_full=False):
+        """
+        Returns start_time and a list with the dubitable antennas.
+        If date is supplied, returns for that date, otherwise now.
+
+        Parameters:
+        ------------
+        date:  something understandable by cm_utils.get_astropytime
+        return_full:  boolean to return the full object as opposed to the csv string
+        """
+
+        at_date = cm_utils.get_astropytime(date)
+        fnd = []
+        for dubi in self.session.query(part_connect.Dubitable):
```

Comment:
This is a little concerning because the sort of search that's happening here should be happening on the DB server side, rather than on our side. The dubitable table is probably not going to get big enough where the performance implications will be a big deal, but it's kind of ugly.

Reply:
The problem is how to handle None, which is what the current stop_gpstime is.  I also wanted to allow dates in the past that have a stop_gpstime.  Given how small the db is for this, it seems clearer to do it this way.

Reply:
One can map the Python values into SQL values by building up the query in pieces, using Python logic to decide whether to add extra terms like the stop time check on top of the start time check. The Librarian does this a lot. But as we've both said, this is unlikely to be a big deal in practice.